README.md

```diff
@@ -1,99 +1,63 @@
-# backstage-operator
-Operator for deploying Backstage for Janus-IDP.
+# Backstage Operator
 
-## Description
-Implementing https://janus-idp.io/docs/deployment/k8s/ procedure
-At first stage CR update does not affect Backstage Objects, just installation (same as Helm)
-TODO: Do we need to continuosly sync the states? Which way if so: from CR to Objects or back or (somehow) back and forth?
-
-Make sure namespace is created.
-
-Local Database (PostgreSQL) is created by default, to disable
-spec: 
-  skipLocalDb: true
-This way third party DB can theorethically be configured. TODO: It just requires some changes in Backstage appConfig (I think), 
-because it only expects either in-container SQLite or MySQL.
-TODO: should we consider using in-container SQLite for K8s deployment as well (single container deployment)?
-
-TODO: POSTGRES_HOST = <name-of the service> , POSTGRES_PORT = <port>[5432] can be delivered to the Backstage 
-Deployment out of Postgres Secret? Indeed, it is not really a secret.
+## The Goal
+The Goal of [Backstage](https://backstage.io) Operator project is creating Kubernetes Operator for configuring, installing and synchronizing Backstage instance on Kubernetes/OpenShift. 
+Primary target is supporting Red Hat's assemblies of Backstage (specifically supporting [dynamic-plugins]()) on OpenShift, such as [Janus-IDP](https://janus-idp.io/) and RHDH but is flexible enough to install any correct Backstage instance (specific Kubernetes deployment with accompanying resources, see [Configuration](#))
```

Comment:
@rm3l 
>Where did you want this to link to?

I think to the dynamic-plugins documentation

Reply:
would that be https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/dynamic-plugins.md ?

```suggestion
Primary target is supporting Red Hat's assemblies of Backstage (specifically supporting [dynamic-plugins]()) on OpenShift, such as [Janus-IDP](https://janus-idp.io/) and RHDH but is flexible enough to install any correct Backstage instance (specific Kubernetes deployment with accompanying resources, see [Configuration](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/dynamic-plugins.md))
```

Reply:
As a general comment, if you reconfigure this as asciidoc instead of markdown, it'll be easier to downstream it to RHDH documentation. cc: @hmanwani-rh  @Gerry-Forde